test/chunking-form/samples/mixed-synthetic-named-exports/_expected/cjs/main.js

```diff
@@ -6,7 +6,7 @@ const d = {
 };
 const foo = 100;
 
-var ns = /*#__PURE__*/Object.freeze(/*#__PURE__*/Object.assign({
```

Comment:
I also removed these "double pure comment" cases since I'm not sure that is necessary for the feature? Otherwise seems odd to potentially have three in a row.

Reply:
Unfortunately all of them are necessary to have an effect. The annotations only annotate the call itself as pure but not its arguments. I readded the annotations and also slightly simplified the logic and merged it with the synthetic export logic.